perception_dataset/fastlabel_to_t4/fastlabel_2d_to_t4_converter.py

```diff
@@ -88,12 +88,27 @@ def convert(self):
                 dataset_name=t4dataset_name,
             )
 
-    def _load_annotation_jsons(self):
+    def _load_annotation_jsons(
+        self, t4_datasets: Optional[List[str]] = None, delimiter: Optional[str] = ""
```

Comment:
```suggestion
        self, t4_datasets: Optional[List[str]] = None, delimiter: Optional[str] = None
```
It looks better to set `delimiter: Optional[str] = None` as a default value.

As far as my experiment, although `.split("")` raises ValueError, `.split(None)` is fine:

```python
>>> "foo".split("")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: empty separator
>>> "foo".split(None)
['foo']
```